general/general_ontology.ttl

```diff
@@ -19,22 +19,6 @@ general:Description a owl:Class ;
     rdfs:label "Description definition for General" ;
     rdfs:comment "Description attributes for all assets"@en .
 
-general:Data a owl:Class ;
```

Comment:
DataShape ist noch im Shacl.
Daher hier bitte wieder aufnehmen (oder in SHACL löschen). Bitte auch an die instance.json denken.